### PR TITLE
[no ticket] update to use 2.13 as default

### PR DESCRIPTION
**Pull request by Qi77Qi**
**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

File: project/Settings.scala [hunk 1]
```diff
@@ -118,7 +118,7 @@ object Settings {
   //common settings for all sbt subprojects
   val commonSettings = commonBuildSettings ++ commonTestSettings ++ List(
     organization := "org.broadinstitute.dsde.workbench",
-    scalaVersion := "2.12.11", //We can't use 2.13 by default yet because some projects are still built for 2.12 only
+    scalaVersion := "2.13.3", //We can't use 2.13 by default yet because some projects are still built for 2.12 only
```
Comment (kyuksel) [inline project/Settings.scala]: Should we remove the comment now?

Comment (Qi77Qi) [reply project/Settings.scala]: updated!

**Review by kyuksel — APPROVED** [review]
🥳

**Comment by kyuksel** [comment]
@Qi77Qi Sorry, forgot to ask during the review: Should we have updated the change log?

**Comment by Qi77Qi** [comment]
@kyuksel nope...it doesn't change anything about the code that's being published

**Pull request merged**